### PR TITLE
Update UniFi Poe port documentation

### PR DESCRIPTION
**Pull request by Kane610**
## Proposed change

Update the documentation to reflect the current implementation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifi.markdown [hunk 1]
```diff
@@ -96,11 +96,9 @@ Clean up clients on the UniFi Network application that has only been associated
 
 Allow control of network access to clients configured in the integration options by adding MAC addresses. Items in this list will have a Home Assistant switch created, using the UniFi Device name, allowing for blocking and unblocking.
 
-### Control clients powered by POE
+### PoE port control
 
-Entities appear automatically for each connected POE client. If no POE client device is in operation, no entity will be visible. Note: UniFi infrastructure devices such as access points and other switches are not (yet) supported, even if they are powered over ethernet themselves.
-
-Note that POE control actually configures the network port of the switch which the client is connected to.
+Provides per port PoE control. Entities are disabled by default. This feature requires admin privileges.
```
Comment (c0ffeeca7) [inline source/_integrations/unifi.markdown]:
```suggestion
Provides per-port PoE control. Entities are disabled by default. This feature requires admin privileges.
```

**Review by c0ffeeca7 — COMMENTED** [review]
tiny tweak

File: source/_integrations/unifi.markdown (hunk 1 above)
Comment (Kane610) [reply source/_integrations/unifi.markdown]: Thanks

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Kane610 :+1:

**Review by frenck — APPROVED** [review]
Thanks, @Kane610 👍 

../Frenck

**Pull request merged**